mpitests/test_adjoint_jacobian.py

```diff
@@ -1044,6 +1044,7 @@ def circuit(params):
         j_gpu = qml.jacobian(qnode_gpu)(params)
 
 
+'''
```

Comment:
Is that to comment the test?

Reply:
Yes, will uncomment it in the sparse Hamiltonian backed adjoint method PR.